.github/workflows/publish-release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Upload template zip to release
+
+on:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: write
+
+jobs:
+  publish:
+    name: Publish zip to release
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Create zip
+        run: |
+          rm -rf /tmp/release
+          mkdir /tmp/release
+          cd src/
+          rsync . -rv --exclude-from=.releaseignore --exclude-from=.gitignore /tmp/release
+          cd /tmp/release
+          zip -r ${{ github.event.repository.name }}.zip .
```

Comment:
Does it add any value to have version number as part of the filename?

Reply:
I thought about adding it, but looking at the github api I thought that this would require us to find the version number of the latest release and use that when downloading the asset (zipfile).
Then again it could possibly make supporting multiple version of template in designer easier.

Reply:
but it seems we would need to get some values either way. I will add the release name to the filename :)